### ISSUE TITLE
Field triggers are missing due to css hidden rule

### ISSUE THREAD

**Issue by javascript-programming** [bryntum/odoo-support#67]
Odoo sets elements with aria-hidden on not visible

Task editor triggers are not visible. This was already solved for the toolbar.
